packages/flutter_tools/lib/src/build_system/targets/icon_tree_shaker.dart

```diff
@@ -24,11 +24,10 @@ const String kIconTreeShakerFlag = 'TreeShakeIcons';
 const bool kIconTreeShakerEnabledDefault = true;
 
 List<Map<String, dynamic>> _getList(dynamic object, String errorMessage) {
-  try {
-    return (object as List<dynamic>).cast<Map<String, dynamic>>();
-  } on TypeError catch (_) {
-    throw IconTreeShakerException._(errorMessage);
+  if (object is List<dynamic>) {
+    return object.cast<Map<String, dynamic>>();
```

Comment:
Can this still throw if it's a `List<String>`?

Reply:
`List<dynamic>` is a `List<Object>`, and `List<String>` is a `List<Object>`. (I checked locally too)